components/property_info.rst

```diff
@@ -0,0 +1,429 @@
+.. index::
+    single: PropertyInfo
+    single: Components; PropertyInfo
+
+The PropertyInfo Component
+==========================
+
+The PropertyInfo component provides the functionality to get information about class properties using metadata of
+popular sources.
+
+Whilst the :doc:`PropertyAccess component</components/property_access/introduction>` allows you to read and write values
+to/from objects and arrays, the PropertyInfo component works solely with class definitions to provide information
+such as data type and visibility about properties within that class.
+
+Similar to PropertyAccess, the PropertyInfo component blurs the lines between class properties such as ``$property``
+and properties defined via accessor and mutator methods such as  ``getProperty()``, ``isProperty()``, ``setProperty()``,
```

Comment:
This list of example method names is not illustrative. I think some actual examples would be better.